docs/testnet/join-testnet-tutorial.md

```diff
@@ -0,0 +1,292 @@
+## Join the Interchain-Security Testnet
+This guide contains the instructions for joining a Interchain-Security Testnet. You can find the instructions to setup a IS-testnet from scratch [here](https://github.com/sainoe/IS-testnet/blob/main/is-testnet-tutorial.md). 
+
+---
+
+### Prerequesites
+- Go 1.18+
+- Interchain-Security binaries
+- jq
+
+#### Install Go
+
+```
+curl -OL https://golang.org/dl/go1.18.4.linux-amd64.tar.gz
+
+sudo tar -C /usr/local -xvf go1.18.4.linux-amd64.tar.gz
+
+export PATH=$PATH:/usr/local/go/bin
+
+export PATH=$PATH:$(go env GOPATH)/bin
+```
+
+---
+
+#### Install Interchain-Security binaries
+
+```
+git clone https://github.com/cosmos/interchain-security.git
+
+cd interchain-security
+
+make install
+```
+
+---
+
+#### Install jq
+```
+#Linux
+apt-get install jq
+
+# OSX
+brew install jq
+```
+
+---
+
+### Run a validator on the Provider chain
+This section will explain you how to setup and run an node in order to participate to the Provider chain as a validator.
+Choose a directory name `<prov-node-dir>` (e.g. `~/provider`) to store the provider chain node files. Along this guide,
+the provider and consumer chain ids used are respectively `provider` and `consumer`.
+
+* *If you have completed the [IS-tesnet tutorial](https://github.com/sainoe/IS-testnet/blob/main/is-testnet-tutorial.md) on the same machine,
+  be sure to use <b>a different node folder</b>!*
+
+__1. Remove any existing directory__  
+
+`rm -rf <prov-node-dir>`  
```

Comment:
ditto: use real folder in commands, e.g., `~/interchain-security-testnet/validator1/provider`